science/nektarpp/Portfile

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               mpi 1.0
+PortGroup               gitlab 1.0
+
+name                    nektarpp
+homepage                https://www.nektar.info/
+
+gitlab.instance         https://gitlab.nektar.info
+gitlab.setup            nektar nektar 5.0.0 v
+
+categories              science
+platforms               darwin
+license                 MIT
+homepage                http://www.nektar.info/
```

Comment:
This line is provided by the gitlab portgroup and can be removed.

Reply:
Ah okay -- I had added this I think in reply to one of the comments above, but now I can't find it so perhaps I'm imagining things. However, the gitlab instance's homepage is just a generic instance, whereas this line adds the main project webpage. Let me know which you would like to keep but my preference would be the latter.

Reply:
Sorry, yes this homepage line (line 17) is fine. The duplicate (line 9) should be removed however.

Reply:
Done!